queries/omen/markets.ts

```diff
@@ -419,7 +427,22 @@ const getMarkets = async (
     fpmm => !marketHasDangerousKeyword(fpmm)
   );
 
```

Comment:
` onst orderBy = params.orderBy as PrimaryOrderBy;`

Let's to something like this to make it easier to read.